token-utility-whitepaper.md

```diff
@@ -1 +1,163 @@
-# Token Utility Whitepaper
+# Token Utility Whitepaper - DRAFT
+
+## Introduction
+A token economy is non-trivial and continues to be an area of much research and debate in the Ethereum community. This proposal document is just that, a proposal which is intended to deliver a viable starting point for an Obscuro token economy. The economic model is expected to evolve over the lifetime of Obscuro via governance proposals.
+Obscuro depends upon a network of intercommunicating Obscuro nodes to achieve its decentralisation, scalability and privacy promises. The Obscuro Mainnet node operators incur operating costs for which they need to be remunerated. This is achieved through a rewards and incentives model for the Obscuro token called OBX. Additionally OBX provides the means for the Obscuro community to vote on change proposals to Obscuro and for ongoing development of the Obscuro ecosystem. Over 50% of OBX is distributed directly to the community so they control the evolution of Obscuro.
+The Obscuro tokenomics have been designed to be circulatory and self-sustaining with no human intervention and no subsidies. Instead decentralised Obscuro smart contracts handle all token movements (transaction fees, rewards and staking).
+Although OBX is not deflationary, token holders are incentivised to stake as Aggregator node operators or as staking delegates by means of a reward. This will lock up OBX in staking pools and reduce the number of OBX in circulation. 
+
+In summary, OBX has four utilities:
+1.	To provide a reward to incentivise node operators  to correctly operate Obscuro nodes and provide a reliable service.
+1.	A medium in which users can pay fees and node operators can pay their stake and receive rewards to cover operational costs.
+1.	A means to vote on proposals for changes to Obscuro via a governance model.
+1.	A means to fund and incentivise the ongoing development of the Obscuro ecosystem via development opportunities, grants and competitions.
```

Comment:
I'm not a lawyer, but is this utility? It's being sold for funding not for payment of services within the network.

Reply:
Anyone buying the tokens are doing so to gain access to the Obscuro network. Buying tokens early on before a public sale means an investor is making a decision to buy OBX at a known price rather than at market price. As a result investors will have the opportunity to run transactions through Obscuro at a discount. This would be advantageous to them if they / their portfolio companies are running a fee-based dapp on top of Obscuro.